internal/cloudapi/v2/compose_test.go

```diff
@@ -812,7 +812,7 @@ func TestGetImageRequests_NoRepositories(t *testing.T) {
 		},
 	}
 	// NOTE: current directory is the location of this file, back up so it can use ./repositories/
-	rr, err := reporegistry.New([]string{"../../../"})
+	rr, err := reporegistry.New([]string{"../../../repositories"}, nil)
```

Comment:
Could we use the embedded repositories from `images` for tests? This will, IMO break once https://github.com/osbuild/osbuild-composer/pull/4554 lands, and if we are not careful and don't rebase the PR that lands as the last, it could pass CI and break `main`.

Reply:
Sure thing, I updated the code to pull in the repos from images now,